documentation/modules/ROOT/partials/modules/tutorial/r_watching-connector-start-up.adoc

```diff
@@ -204,6 +204,10 @@ Step 7 releases the global write lock just 0.3 seconds after acquiring it,
 and Step 8 reads all of the rows in each of the tables and reports the time taken and number of rows found.
 In this case, the connector completed its consistent snapshot in just 0.38 seconds.
 
+:link-prefix: xref
```

Comment:
Hum, that's interesting, that's already configured in a central place already, no?

Reply:
Argh...I add those temporarily to test something and neglected to remove them. I just pushed a new commit to remove them.